### ISSUE TITLE
Question: Why isn't JMAP getting much traction?

### ISSUE THREAD

**Issue by Richie765** [jmapio/jmap#340]
I've been looking at JMAP with interest over the last couple of years, but I see veeeeeery little adoption. There are exactly 0 mainstream clients and exactly 1 of the major email services (fastmail) support it. Basically it does not go any further than the creators and initial adopters.

What could be the reason for it? One reason I could think of is the lack of a IMAP <-> JMAP proxy (let's not consider that Perl abomination here). I think a good working Node based server/proxy is really the missing piece here.

Not here to criticize, but I think it's a question you guys must have been pondering for a while now as well and I'm curious what are your thoughts and if anything is going on to make it finally happen or not.

**Comment by neilj** [comment]
Well, two things here really:

1. The average time for protocols to go from standardisation to wide-spread adoption is probably much longer than you think, especially for larger organisations that move more slowly. For example, IMAP4 rev 1 (the most recent base IMAP standard) was published as [RFC 3501](https://tools.ietf.org/html/rfc3501) in March 2003. Gmail was released in 2004 but didn't add IMAP support until [October 2007](https://en.wikipedia.org/wiki/History_of_Gmail). Meanwhile, it took until [September 2013](https://web.archive.org/web/20130912204726/http://blogs.office.com/b/microsoft-outlook/archive/2013/09/12/outlook-com-now-with-imap.aspx) for Outlook.com (neé Hotmail) to add IMAP support.
2. Just because you don't see full support announced publicly doesn't mean that people aren't working on it.

Basically, it's early days. I'd love things to happen faster, but we do see things progressing.

**Comment by Richie765** [comment]
Ah, is that it. Thanks. I admire your patience though.
